### ISSUE TITLE
Detail of experiment

### ISSUE THREAD

**Issue by Albertvicient** [AdaptiveMotorControlLab/CEBRA-demos#12]
Hi, your work is very excellent. I want to learn more about the detail of the experiment. Could you tell me the detail of Forelimb movement behavior in a primate experiment. How to divide the training set and test set in this experiment.

**Comment by MMathisLab** [comment]
Hi @Albertvicient, thanks! We did not collect the data, but it can be found here: https://elifesciences.org/articles/48198
